api/src/uvisor-input.S

```diff
@@ -82,10 +92,28 @@ uvisor_config:
     .long __uvisor_sram_start;
     .long __uvisor_sram_end;
 
+    /* Privileged handlers */
+    /* These have the same name as the CMSIS interrupt handlers on purpose, so
+     * we can re-use those already defined handlers and avoid modifying
+     * pre-existing applications for uVisor compatibility. */
+    .long SVC_Handler
+    .long PendSV_Handler
+    .long SysTick_Handler
+
 __uvisor_mode:
     /* uvisor default mode - user can override weak reference */
     .long 0
 
+    /* Default privileged handlers */
+SVC_Handler:
+    .long 0
```

Comment:
@Patater why are the default handler be not an endless loop, but .long? If I correctly understand the data at line 99 will contain a pointer to the .long in 109.